core/database_api.php

```diff
@@ -852,10 +867,6 @@ function db_get_table_list() {
 function db_update_blob( $p_table, $p_column, $p_val, $p_where = null ) {
 	global $g_db, $g_db_log_queries, $g_queries_array;
 
-	if( !db_is_oracle() ) {
-		return false;
-	}
-
```

Comment:
Why remove this safety check ? The function should return false for DBs not using BLOBs.

Check PHPDoc, update accordingly.

Reply:
The db_update_blob() function is called everywhere only after if( db_is_oracle() ), there is no point in doing double check inside the function, especially since it is now also needed in the case of db_is_sqlite().

I've changed the description accordingly.